app/v2/models/request/common/ukFhlPieProperty/UkFhlPieIncome.scala

```diff
@@ -22,19 +22,19 @@ import v2.models.request.common.ukPropertyRentARoom.UkPropertyIncomeRentARoom
 
 case class UkFhlPieIncome(periodAmount: Option[BigDecimal], taxDeducted: Option[BigDecimal], ukRentARoom: Option[UkPropertyIncomeRentARoom])
 
-object UkFhlPieIncome{
+object UkFhlPieIncome {
 
- implicit val reads: Reads[UkFhlPieIncome] =
-  (
-    (__ \ "periodAmount").readNullable[BigDecimal] and
-    (__ \ "taxDeducted").readNullable[BigDecimal] and
-    (__ \ "rentARoom").readNullable[UkPropertyIncomeRentARoom]
-  )(UkFhlPieIncome.apply _)
+  implicit val reads: Reads[UkFhlPieIncome] =
```

Comment:
Just a suggestion - totally optional! - but could I suggest calling these `mtdReads` and `downstreamWrites`?
(And doing that as a general rule). It makes their intended use much clearer.